tutorial/PinCell/pin_cell.i

```diff
@@ -0,0 +1,136 @@
+global_temperature=600
+
+[GlobalParams]
+  num_groups =2
+  base_file = 'PinXS.h5'
+  group_fluxes = ' group1  group2'
+  num_precursor_groups = 6
+  use_exp_form = false
+  temperature = ${global_temperature}
+[../]
+
+[Mesh]
+  file = pin_cell.e
+[../]
+
+[Problem]
+  type = FEProblem
+  coord_type = XYZ
+[]
+
+[Nt]
+  var_name_base = group
+  create_temperature_var = false
+  eigen = true
+  account_delayed = 0
+  sss2_input = 1
+[]
+
+[Materials]
+  [./F]
+    type = H5MoltresMaterial
+    block = 1
+    material_key = 'F'
+    interp_type = 'NONE'
+    temperature = 600
+  [../]
+  [./C]
+    type = H5MoltresMaterial
+    block = 3
+    material_key = 'C'
+    interp_type = 'NONE'
+    temperature = 600
+  [../]
+  [./W]
+    type = H5MoltresMaterial
+    block = 4
+    material_key = 'W'
+    interp_type = 'NONE'
+    temperature = 600
+  [../]
+[]
+
+[Executioner]
+   #type = NonlinearEigen
```

Comment:
Try not to commit commented code. 
Delete the lines, don't comment them out.
You can always see what you used to do by going back in time with version  control.